Patch-Source: https://github.com/misterdanb/avizo/pull/32

From 3b47b0fc1e3278d1a0e437535155df4d625ea4d2 Mon Sep 17 00:00:00 2001
From: Jakub Jirutka <jakub@jirutka.cz>
Date: Sat, 15 Jan 2022 02:07:53 +0100
Subject: [PATCH] Fix bug in rgba() function - incorrect number type/range

This bug was introduced in 5f41024730f74076ec9324551cc650b845d026bc
by myself.

This function is used for the default colour values. Since there are
two, one of them is white and other black, it currently doesn't affect
the application. That's also why I didn't notice it until I started
changing the defaults.
---
 src/avizo_service.vala | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/avizo_service.vala b/src/avizo_service.vala
index 7587ad5..1c7699a 100644
--- a/src/avizo_service.vala
+++ b/src/avizo_service.vala
@@ -268,12 +268,12 @@ public class AvizoService : GLib.Object
 	}
 }

-Gdk.RGBA rgba(double red, double green, double blue, double alpha)
+Gdk.RGBA rgba(int red, int green, int blue, double alpha)
 {
 	var o = Gdk.RGBA();
-	o.red = red;
-	o.green = green;
-	o.blue = blue;
+	o.red = red / 255.0;
+	o.green = green / 255.0;
+	o.blue = blue / 255.0;
 	o.alpha = alpha;
 	return o;
 }
